### PR TITLE
Add software development oriented bepo layout

### PR DESCRIPTION
**Pull request by lebastaq**
The bepo layout is an ergonomic layout for french language (like dvorak).
While the original layout enables easy access to french-specific letters, it is not tailored for easy english writing and software development.

This is a modified version for the XD75, with: 
* Improved access to frequently used keys in software development
* Modified layout for english input
* Easy access to layers with space and enter key layer shifts
* Vim-style arrow keys 
* Easy access to page up/down, home and end

The central column is not populated with key codes, as I do not personally use it. It requires my hands to move too much, and I intend to later move to a smaller layout.

### CODE REVIEW

File: keyboards/xd75/keymaps/developper_bepo/config.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/xd75/keymaps/developper_bepo/config.h]: Could this line and everything after it be replaced with `#pragma once`?

File: keyboards/xd75/keymaps/developper_bepo/keymap.c [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "xd75.h"
```
Comment (drashna) [inline keyboards/xd75/keymaps/developper_bepo/keymap.c]: Could this be replaced with `#include QMK_KEYBOARD_H`

File: keyboards/xd75/keymaps/developper_bepo/rules.mk [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/xd75/keymaps/developper_bepo/rules.mk]: This block isn't needed, and is a holdover from the old make formatting.  Could you remove it?

**Comment by lebastaq** [comment]
Hey,
Thanks for the comments - I made the requested changes

**Comment by lebastaq** [comment]
I also updated the traditional bepo layout with the same changes you requested, so that it is more up-to-date

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**